web-ui/src/pages/FeedbackRequestPage.jsx

```diff
@@ -145,24 +187,18 @@ const FeedbackRequestPage = () => {
             <div>
               <Link
                 className={`no-underline-link ${activeStep <= 1 ? 'disabled-link' : ''}`}
-                to={getFeedbackArgs(activeStep - 1)}
+                to={`?step=${activeStep - 1}`}
```

Comment:
This now erases the selected template, recipients, dates, etc. from the URL when going back. Is this intended behavior?

Reply:
Additionally, if you select a template and then recipients, and then hit the back button, the page breaks and throws an error in the console: `Uncaught Error: FeedbackRequestPage(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.`

Reply:
This is a bug...I would argue for simply doing`history.goBack()`